src/components/views/UserPage.js

```diff
@@ -138,18 +111,25 @@ const UserPage = () =>{
                             <td> Birthday: </td>
                             <td className="userPage td"> {displayDate(user.birthday)} </td>
                         </tr>
-                    </tbody>
+                        <tr>
+                            <td colspan="2">
+                                <Button
+                                    className="invert"
+                                    width="100%"
+                                    onClick={() => goToEdit()}
+                                >
+                                    Edit Profile
+                                </Button>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>&nbsp;</td>
```

Comment:
What does this do? (What's nbsp?)

Reply:
it creates an empty cell in the table, only needed for design purposes, could be solved with a className but as the design is not final i decided to solve it like this..